views/account.ejs

```diff
@@ -187,4 +214,32 @@
     <% } %>
 
 </div>
+
+<script>
+    document.addEventListener('DOMContentLoaded', function () {
+        
+        const collapseElements = document.getElementsByClassName('collapse');
+        for (let elem of collapseElements) {
```

Comment:
attention, ce code doit tourner dans tous les browsers sans transpilation, donc il faut éviter le fancy JS recent. 
 - for .. of : pas supporté par IE
 - arrow functions (`event => ...`) : pas supporté par IE, et sur les principaux browsers avant 2015-2016 
 
(pour voir si une feature est supportée par un browser, google par exemple "arrow function browser support". MDN et caniuse.com sont des bonnes ressources. Pour l'exemple : https://caniuse.com/arrow-functions)

Reply:
C'est juste IE 11, on n'a pas fait de choix, est-ce que l'on supporte IE 11 ? on peut choisir de ne pas supporter ou de laisser ouvert tous les accordons.

L'app de secretariat reste interne à la communauté.